packages/nitro-protocol/benchmark/signState.ts

```diff
@@ -0,0 +1,31 @@
+import {Wallet, constants} from 'ethers';
+import {AddressZero} from 'ethers/constants';
+import _ from 'lodash';
+
+import {signState, fastSignState} from '../src/signatures';
+import {State} from '../src';
+
+async function benchmark() {
```

Comment:
It's be nice to run these benchmarks in circle. We'd then have a historical reference, and a more consistent environment.